core/src/main/java/org/apache/struts2/StrutsConstants.java

```diff
@@ -346,4 +346,9 @@
 
     /** See {@link com.opensymphony.xwork2.config.impl.AbstractMatcher#appendNamedParameters */
     public static final String STRUTS_MATCHER_APPEND_NAMED_PARAMETERS = "struts.matcher.appendNamedParameters";
+
+    public static final String STRUTS_CHAINING_COPY_ERRORS = "struts.chaining.copyErrors";
```

Comment:
I guess according to https://issues.apache.org/jira/browse/WW-5000?focusedCommentId=17257465&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17257465 the removal of `xwork.` is intentional?

struts.xwork.chaining.copyErrors gets struts.chaining.copyErrors

Reply:
Yes, this is a major release so I think it's a good time to change this. XWork doesn't exist as a seperated artefact anymore.